docs/src/code_examples/sandbox_example.ts

```diff
@@ -0,0 +1,178 @@
+import {
+  AztecRPC,
+  L2BlockL2Logs,
+  PrivateKey,
+  createAztecRpcClient,
+  createDebugLogger,
+  getSchnorrAccount,
+  mustSucceedFetch,
+} from "@aztec/aztec.js";
+import { PrivateTokenContract } from "@aztec/noir-contracts/types";
+
+// docs:start:index.ts
+////////////// CREATE THE CLIENT INTERFACE AND CONTACT THE SANDBOX //////////////
+const logger = createDebugLogger("private-token");
+const sandboxUrl = "http://localhost:8080";
+
+const aztecRpc = createAztecRpcClient(sandboxUrl, mustSucceedFetch);
+
+const nodeInfo = await aztecRpc.getNodeInfo();
+
+logger("Aztec Sandbox Info ", nodeInfo);
+// docs:end:index.ts
+
+// docs:start:Accounts
+////////////// CREATE SOME ACCOUNTS WITH SCHNORR SIGNERS //////////////
+// Creates new accounts using an account contract that verifies schnorr signatures
+// Returns once the deployment transactions have settled
+const createSchnorrAccounts = async (
+  numAccounts: number,
+  aztecRpc: AztecRPC
+) => {
+  const accountManagers = Array(numAccounts)
+    .fill(0)
+    .map((x) =>
+      getSchnorrAccount(
+        aztecRpc,
+        PrivateKey.random(), // encryption private key
+        PrivateKey.random() // signing private key
+      )
+    );
+  return await Promise.all(
+    accountManagers.map(async (x) => {
+      await x.waitDeploy({});
+      return x;
+    })
+  );
+};
+
+// Create 2 accounts and wallets to go with each
+logger(`Creating accounts using schnorr signers...`);
+const accounts = await createSchnorrAccounts(2, aztecRpc);
+
+////////////// VERIFY THE ACCOUNTS WERE CREATED SUCCESSFULLY //////////////
+
+const [alice, bob] = (
+  await Promise.all(accounts.map((x) => x.getCompleteAddress()))
+).map((x) => x.address);
+
+// Verify that the accounts were deployed
+const registeredAccounts = await aztecRpc.getAccounts();
+for (const [account, name] of [
+  [alice, "Alice"],
+  [bob, "Bob"],
+] as const) {
+  if (registeredAccounts.find((acc) => acc.equals(account))) {
+    logger(`Created ${name}'s account at ${account.toShortString()}`);
+    continue;
+  }
+  logger(`Failed to create account for ${name}!`);
+}
+// docs:end:Accounts
+
+// docs:start:Deployment
+////////////// DEPLOY OUR PRIVATE TOKEN CONTRACT //////////////
+
+// Deploy a private token contract, create a contract abstraction object and link it to the owner's wallet
+// The contract's constructor takes 2 arguments, the initial supply and the owner of that initial supply
+const initialSupply = 1_000_000;
+logger(
+  `Deploying private token contract minting an initial ${initialSupply} tokens to Alice...`
+);
+const tokenContractTx = PrivateTokenContract.deploy(
+  aztecRpc,
+  initialSupply, // the initial supply
+  alice // the owner of the initial supply
+).send();
```

Comment:
If you pass in the wallet instead of the aztecRpc, you can call `deployed()` to get the instance. Note that this only makes sense if we adapt this flow to go with a single wallet.